src/img/logo-dark.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="213" height="20" viewBox="0 0 213 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is there no logos in the code base already?

Reply:
not this specific ones, this is new as per figma